tests/acceptance/pageObjects/phoenixPage.js

```diff
@@ -76,23 +79,27 @@ module.exports = {
     assertNotificationIsPresent: async function (numberOfNotifications, expectedNotifications) {
       const notifications = await this.getNotifications()
       assert.strictEqual(notifications.length, numberOfNotifications)
+      const promises = []
       for (const element of expectedNotifications) {
-        const isPresent = notifications.includes(element.title)
-        this.assert.ok(isPresent)
+        const isPresent = notifications.includes(userSettings.replaceInlineCode(element.title))
+        promises.push(this.assert.ok(isPresent))
       }
+      return Promise.all(promises)
     },
     /**
      * Perform accept action on the offered shares in the notifications
      */
     acceptAllSharesInNotification: async function () {
       const notifications = await this.getNotifications()
+      await this.toggleNotificationDrawer()
       for (const element of notifications) {
         const acceptShareButton = util.format(this.elements.acceptSharesInNotifications.selector, element)
         await this
           .useXpath()
+          .initAjaxCounters()
           .waitForElementVisible(acceptShareButton)
           .click(acceptShareButton)
-          .waitForAjaxCallsToStartAndFinish()
+          .waitForOutstandingAjaxCalls()
```

Comment:
I think, this was added to increase waiting as it was failing quite. Talk with @jasson99 about it.

Reply:
Also, please write a comment, describing this.